pkg/manager/member/pd_member_manager_test.go

```diff
@@ -1088,7 +1088,7 @@ func TestGetPDConfigMap(t *testing.T) {
 					Labels: map[string]string{
 						"app.kubernetes.io/name":       "tidb-cluster",
 						"app.kubernetes.io/managed-by": "tidb-operator",
-						"app.kubernetes.io/instance":   "",
+						"app.kubernetes.io/instance":   "foo",
```

Comment:
Use const vars in label package to replace these keys?

Reply:
I think both make sense, by not using the const var, the correctness of the `labelKey` itself is also tested, it just depends on the purpose of the UTs